lib/active_merchant/billing/affirm.rb

```diff
@@ -0,0 +1,49 @@
+require 'affirm'
+
+module ActiveMerchant #:nodoc:
```

Comment:
Why are we using an active merchant gateway instead of a `Spree::Gateway`? Ala https://github.com/solidusio/solidus_braintree/blob/master/app/models/solidus/gateway/braintree_gateway.rb#L3